### PR TITLE
chore(litmus-agent): adding litmus-agent helm chart

### PR DESCRIPTION
**Pull request by ispeakc0de**
Signed-off-by: Shubham Chaudhary <shubham.chaudhary@harness.io>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] DCO signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md

### CODE REVIEW

**Review by Jonsy13 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -106,6 +106,55 @@ chaos-litmus-mongo-0                     1/1     Running            0          7
...
+helm install litmus-agent litmuschaos/litmus-agent \
+--namespace litmus --create-namespace \
+--set "AGENT_NAME=helm-agent" \
+--set "AGENT_DESCRIPTION=My first agent deployed with helm !" \
+--set "LITMUS_URL=https://chaos-center.domain.com" \ # FOR REMOTE AGENT (INGRESS)
+--set "LITMUS_URL=http://litmusportal-frontend-service.litmus.svc.cluster.local:9091" \ # FOR SELF AGENT (SVC)
+--set "LITMUS_BACKEND_URL=http://litmusportal-server-service.litmus.svc.cluster.local:9002" \ # FOR SELF AGENT (SVC)
+--set "LITMUS_USERNAME=admin" \
+--set "LITMUS_PASSWORD=litmus" \
+--set "LITMUS_PROJECT_ID=69395cb3-0231-4262-8990-78056c8adb4c" \
```
Comment (Jasstkn) [inline README.md:154]: is it necessary to hardcode project id here?

File: charts/litmus-agent/charts/chaos-exporter/templates/_helpers.tpl [hunk 2]
```diff
@@ -0,0 +1,63 @@
+{{/*
```
Comment (Jasstkn) [inline charts/litmus-agent/charts/chaos-exporter/templates/_helpers.tpl:1]: why do we need an empty NOTES.txt?

File: charts/litmus-agent/charts/chaos-exporter/templates/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: http
+              containerPort: 8080
+              protocol: TCP
+          env:
+            - name: TSDB_SCRAPE_INTERVAL
+              value: '10'
```
Comment (Jasstkn) [inline charts/litmus-agent/charts/chaos-exporter/templates/deployment.yaml:42]: make it configurable?

File: charts/litmus-agent/charts/chaos-exporter/templates/service-monitor.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+{{- if .Values.serviceMonitor.enabled }}
```
Comment (Jasstkn) [inline charts/litmus-agent/charts/chaos-exporter/templates/service-monitor.yaml:1]: I would suggest to make this template more flexible. Good example is here: https://github.com/litmuschaos/litmus-helm/blob/master/charts/litmus-core/templates/exporter-servicemonitor.yaml

File: charts/litmus-agent/charts/chaos-exporter/Chart.yaml [hunk 5]
```diff
@@ -0,0 +1,20 @@
+apiVersion: v2
```
Comment (Jasstkn) [inline charts/litmus-agent/charts/chaos-exporter/Chart.yaml:1]: why it's empty? as well as others in charts/ folder.

File: README.md (hunk 1 above)
Comment (ispeakc0de) [reply README.md:154]: Project id is a mandatory field. Alternative is I can provide a placeholder for project id

**Pull request merged**